content/sensu-go/5.17/reference/secret-providers.md

```diff
@@ -0,0 +1,300 @@
+---
+title: "Secrets providers"
+linkTitle: "Secrets Providers"
+description: "Sensu's secrets management capability allows you to avoid exposing secrets in your Sensu configuration. Read the reference to obtain secrets from one or more external secrets management providers and support references to external secrets in your Sensu configuration."
+weight: 148
+version: "5.17"
+product: "Sensu Go"
+menu: 
+  sensu-go-5.17:
+    parent: reference
+---
+
+- [Secrets providers specification](#secrets-providers-specification)
+  - [Top-level attributes](#top-level-attributes) | [Metadata attributes](#metadata-attributes) | [Spec attributes](#spec-attributes)
+- [Secrets providers examples](#secrets-providers-examples)
+
+**COMMERCIAL FEATURE**: Access the Env and Vault datatypes for secrets providers in the packaged Sensu Go distribution.
+For more information, see [Get started with commercial features][1].
+
+Sensu's secrets management eliminates the need to expose secrets (e.g. usernames and passwords) in your Sensu configuration.
+With Sensu's secrets management, you can obtain secrets from one or more external secrets providers, refer to external secrets, and consume secrets via backend [environment variables][4].
+
+Only Sensu backends have access to request [secrets][9] from a secrets provider.
```

Comment:
Should this link be to the secrets reference rather than the secrets providers reference?

Reply:
Yes -- good catch, thank you!